footer.php

```diff
@@ -9,73 +9,75 @@
 // Ignore the issues of this file since that's just a copied HTML placeholder.
 // @codingStandardsIgnoreFile
 ?>
+<?php if ( ! is_search() ): ?>
 <!-- Footer -->
-        <footer class="travel-footer overflow-hidden">
+	<footer class="travel-footer overflow-hidden">
 
-            <div class="relative bg-black">
+		<div class="relative bg-black">
 
-                <!-- Angle -->
-                <div class="travel-footer-angle-block absolute top-0 bottom-0 right-0 xs-hide sm-hide"></div>
-                <div class="travel-footer-angle absolute xs-hide sm-hide"></div>
-                <!--/ Angle -->
+			<!-- Angle -->
+			<div class="travel-footer-angle-block absolute top-0 bottom-0 right-0 xs-hide sm-hide"></div>
+			<div class="travel-footer-angle absolute xs-hide sm-hide"></div>
+			<!--/ Angle -->
 
-                <!-- Right column -->
-                <div class="travel-newsletter-signup">
-                    <header class="max-width-3 mx-auto px1 md-px2">
-                        <h4 class="travel-footer-right-column-heading travel-spacing-none h2 mb3 blue">Want travel deals<br>straight to your inbox?</h4>
-                        <div class="h4 bold mb1">Sign up to our newsletter</div>
-                    </header>
-                    <div class="relative">
-                        <div class="travel-footer-input-bg bg-black absolute right-0 left-0 md-hide lg-hide"></div>
-                        <div class="max-width-3 mx-auto px1 md-px2 relative">
-                            <div class="travel-input-group flex items-center col-12 rounded travel-shadow">
-                                <input class="travel-input travel-input-big block col-12 flex-auto rounded-left" type="text" name="email" placeholder="Enter your email">
-                                <span class="travel-input-group-sep travel-border-gray relative z1 block"></span>
-                                <button type="button" class="travel-link travel-input-big nowrap rounded-right">
-                                    Sign up now
-                                </button>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <!--/ Right column -->
+			<!-- Right column -->
+			<div class="travel-newsletter-signup">
+				<header class="max-width-3 mx-auto px1 md-px2">
+					<h4 class="travel-footer-right-column-heading travel-spacing-none h2 mb3 blue">Want travel deals<br>straight to your inbox?</h4>
```

Comment:
This untranslated text is probably alright for now, as this template is a placeholder.